crates/holochain/src/conductor/conductor.rs

```diff
@@ -2963,6 +2962,103 @@ mod misc_impls {
             Ok(action_hash)
         }
 
+        /// Get capability grant info for a set of App cells including revoked capabality grants
```

Comment:
This is good. I think most of the time though, the request is mostly going to be for cap grants that are still valid. Perhaps we could include a filter parameter on the request that is passed to this function?